streetcrm/admin.py

```diff
@@ -552,8 +551,24 @@ def advanced_search_do(self, request, form, export):
                                "result_count": count_set['result_count']
             }
         else:
+            search_params = []
+            for key, value in data.items():
+                if isinstance(value, datetime):
+                    search_params.append('{} = {}'.format(key, value.strftime('%Y-%m-%d')))
+                elif isinstance(value, str) and len(value):
+                    search_params.append('{} = {}'.format(key, value))
+                elif isinstance(value, bool) and value:
+                    search_params.append(key)
+                elif hasattr(value, '__iter__') and len(value):
+                    search_params.append('{} = {}'.format(key, ','.join([str(v) for v in value])))
+                elif hasattr(value, 'pk'):
+                    search_params.append('{} = {}'.format(key, str(value)))
+
+            search_header = ', '.join(search_params)
+
             results_package = {"form": form,
                                "search_results": results,
+                               "search_header": search_header,
```

Comment:
I'm really passing on a comment from @cecilia-donnelly here, expanded slightly by me: wouldn't `export_search` be the right place to assemble this new search_header?  IOW, instead of assembling it in `advanced_search_do` (which doesn't know whether or not its results will ever be exported), do it in `export_search`, where by definition we know that the export is happening.